cmd/syncer/main.go

```diff
@@ -81,13 +80,15 @@ func main() {
 	var controlPlaneConfigSecretNamespace string
 	var syncedResources arrayFlags
 	var controlPlaneNS string
+	var dataPlaneNS string
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&controlPlaneConfigSecretName, "control-plane-config-name", "control-plane-cluster", "The name of the secret with the control plane client configuration")
 	flag.StringVar(&controlPlaneConfigSecretNamespace, "control-plane-config-namespace", "mctc-system", "The namespace containing the secret with the control plane client configuration")
 	flag.Var(&syncedResources, "synced-resources", "A list of GVRs to sync (e.g. ingresses.v1.networking.k8s.io)")
 	flag.StringVar(&controlPlaneNS, "control-plane-namespace", "mctc-tenant", "The name of the upstream namespace to sync resources from")
+	flag.StringVar(&dataPlaneNS, "data-plane-namespace", os.Getenv("DATAPLANE_NAMESPACE"), "The namespace in the data plane to sync resources to")
```

Comment:
@philbrookes Should this default to something if `data-plane-namespace` isn't explicitly passed and DATAPLANE_NAMESPACE is not set? Currently it tries to create a "" ns.  fyi, we did this before in glbc see https://github.com/kcp-dev/kcp-glbc/blob/main/cmd/kcp-glbc/main.go#L92-L101